content/300-accelerate/600-faq.mdx

```diff
@@ -125,3 +125,27 @@ In the rare event of a service disruption, falling back to a direct connection w
 If there is a service disruption, it's recommended to verify on the [status page](https://pris.ly/data-platform-status). You can reach out to one of Prisma's [support channels](/platform/support) for assistance.
 
 > **Note:** Additionally, it's worth noting that some edge function runtime environments may not support direct connections with Prisma ORM. For further details, refer to our [Edge functions documentation](/orm/prisma-client/deployment/edge/overview).
+
+Sure! Here's the revised FAQ item:
```

Comment:
Why is this here? Forget to remove the chatgpt part?

Reply:
My bad.